src/main/java/seedu/address/logic/commands/MarkTaskCommand.java

```diff
@@ -0,0 +1,108 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PROJECT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TASK;
+
+import java.util.List;
+import java.util.logging.Logger;
+
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.project.Project;
+import seedu.address.model.task.Task;
+import seedu.address.model.task.TaskList;
+
+
+/**
+ * Marks a task as completed in TaskHub.
+ */
+public class MarkTaskCommand extends Command {
+
+    public static final String COMMAND_WORD = "markT";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Marks a task as completed in TaskHub.\n"
+            + "Parameters: " + PREFIX_PROJECT + "PROJECT_INDEX " + PREFIX_TASK + "TASK_INDEXES\n"
+            + "PROJECT_INDEX must be one positive integer and "
+            + "TASK_INDEXES must be one or more positive integers, separated by a space between each INDEX\n"
+            + "Example: " + COMMAND_WORD + " " + PREFIX_PROJECT + "2 " + PREFIX_TASK + "1 2 4";
+
+    public static final String MESSAGE_TASKS_MARKED_SUCCESSFULLY = "%1$d task(s) marked as completed.";
```

Comment:
I think it may be more user-friendly if we were to add the project index in the successful message as well. This will make the user better aware of the change (on top of the visual GUI change). Similar to how humans usually want to "double-confirm" things.

Reply:
I do agree, however, in this case I find that immediately upon the completion of the command, the full project list is shown (similar to the behaviour of the other project commands). If the project's index in the filtered project list differs from that in the full project list, this could potentially cause confusion for the user.

Reply:
As discussed, have amended the success message to include the project name instead.